src/Pinch/Protocol.hs

```diff
@@ -14,6 +14,7 @@
 module Pinch.Protocol
     ( Protocol(..)
     , deserializeValue
+    , deserializeValue'
```

Comment:
This is already exported because `Protocol(..)` is exported.

http://hackage.haskell.org/package/pinch-0.3.4.1/docs/Pinch-Protocol.html#v:deserializeValue-39-

Reply:
Oh, right. Fixed.